docs/prow/authorization.md

```diff
@@ -2,10 +2,45 @@
 
 ## Required GCP Permissions
 
-Every Prow clsuter should be deployed in its own GCP project. You need an owner access to deploy Prow and configure it properly.
+Every Prow cluster should be deployed in its own GCP project. You need an owner access to deploy Prow and configure it properly. In addition you need following service accounts created:
```

Comment:
In order to deploy a Prow cluster, configure the following service accounts in the GCP project you own.

Reply:
done